### PR TITLE
Feature/20230309 julian

### PR DESCRIPTION
**Pull request by Julian0701**
### DEVELOP
- [x] position open modal
- [ ] history position item

### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component: 0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/user_overview/user_overview.tsx [hunk 1]
```diff
@@ -44,20 +44,20 @@ const UserOverview = ({
...
-            <div className="whitespace-nowrap p-2 text-sm text-lightGray4">Available</div>
-            <div className="whitespace-nowrap p-2 text-xs">{depositAvailable} USDT</div>
+            <div className="whitespace-nowrap p-1 text-sm text-lightGray4">Available</div>
+            <div className="whitespace-nowrap p-1 text-xs">{depositAvailable} USDT</div>
           </div>
 
           <div className="">
-            <div className="whitespace-nowrap p-2 text-sm text-lightGray4">M. Margin</div>
-            <div className="whitespace-nowrap p-2 text-xs">{marginLocked} USDT</div>
+            <div className="whitespace-nowrap p-1 text-sm text-lightGray4">M. Margin</div>
```
Comment (Julian0701) [inline src/components/user_overview/user_overview.tsx:54]: 整理語言檔

File: src/components/user_overview/user_overview.tsx [hunk 2]
```diff
@@ -44,20 +44,20 @@ const UserOverview = ({
...
+            <div className="whitespace-nowrap p-1 text-xs">{marginLocked} USDT</div>
           </div>
 
           <div className="">
-            <div className="whitespace-nowrap p-2 text-sm text-lightGray4">PNL</div>
-            <div className="whitespace-nowrap p-2 text-xs">{profitOrLossAmount} USDT</div>
+            <div className="whitespace-nowrap p-1 text-sm text-lightGray4">PNL</div>
```
Comment (Julian0701) [inline src/components/user_overview/user_overview.tsx:59]: 整理語言檔

File: src/components/user_overview/user_overview.tsx [hunk 3]
```diff
@@ -44,20 +44,20 @@ const UserOverview = ({
       <div className="mt-5">
         {/* <div className="my-auto h-px w-full rounded bg-white/50"></div> */}
 
-        <div className="flex space-x-10 text-center lg:hidden">
+        <div className="flex justify-center space-x-10 text-center lg:hidden">
           <div className="">
-            <div className="whitespace-nowrap p-2 text-sm text-lightGray4">Available</div>
-            <div className="whitespace-nowrap p-2 text-xs">{depositAvailable} USDT</div>
+            <div className="whitespace-nowrap p-1 text-sm text-lightGray4">Available</div>
```
Comment (Julian0701) [inline src/components/user_overview/user_overview.tsx:49]: 整理語言檔

**Review by Julian0701 — COMMENTED** [review]
修正錯誤

**Review by Luphia — APPROVED** [review]
- no bug found

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**